styleguide/source/_patterns/01-molecules/06-components/20-testimonial/_testimonial.scss

```diff
@@ -0,0 +1,64 @@
+.testimonial {
+  position: relative;
+  width: 100%;
+  max-width: 1200px;
+  margin-bottom: 24px;
+
+  @include breakpoint($bp-med) {
+    display: flex;
+    flex-direction: row-reverse;
+    align-items: stretch;
+
+    &.testimonial-left {
+      flex-direction: row;
+    }
+  }
+}
+
+.testimonial_image {
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+  min-height: calc(((100vw - 40px) / 3) * 2);
+
+  @include breakpoint($bp-med) {
+    min-height: calc(((100vw -  60px) / 2) / 3 * 2);
```

Comment:
I like this solution. Is this what you were thinking? I vertically centered the image. 

![3629-1](https://user-images.githubusercontent.com/12160398/27499500-be0217d6-5829-11e7-9d35-74d89a093a83.gif)